packages/ez-react/src/components/Areas.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, SVGAttributes, useMemo } from 'react';
+import { AreaConfig, AreaData, MarkerConfig } from 'eazychart-core/src/types';
+import { Point } from '@/components/shapes/Point';
+import { Points } from '@/components/Points';
+import { useColorScale } from '@/components/scales/ColorScale';
+import { Area } from './shapes/Area';
+import { LinePath } from './shapes/LinePath';
+
+export interface AreasProps extends SVGAttributes<SVGGElement> {
+  xDomainKey: string;
+  yDomainKey: string;
+  area?: AreaConfig;
+  marker?: MarkerConfig;
+}
+
+export const Areas: FC<AreasProps> = ({
```

Comment:
```suggestion
export const Area: FC<AreaProps> = ({
```
I think this should be in singular form (also filename).